external/grpc/README.md

```diff
@@ -0,0 +1,110 @@
+# gRPC framework for TizenRT.
+TizenRT features the gRPC framework from the opensource project [gRPC](https://github.com/grpc/grpc), and is versioned at 1.9.
+TizenRT's version of gRPC supports only C++ binding, so it is expected that the developer should write C++ applications in order to 
+use gRPC. In this document, we cover two main aspects of porting gRPC to TizenRT, mainly details on how to build gRPC library, and how
+to develop, build and run applications that use gRPC.
+
+## Build gRPC in TizenRT
+The source files for gRPC are located at `external/grpc`, and the associated header files are located at 
+`external/include/grpc` folders. To build gRPC in TizenRT, you need to enable CONFIG_GRPC defined in `external/grpc/Kconfig.protocol`, as shown below:
+```
+config GRPC
```

Comment:
You had better give the structure on menuconfig, not source code level of Kconfig.